httpapi/router.go

```diff
@@ -23,7 +23,7 @@ import (
 // Router returns the http mux for the server.
 // @title Centrifuge OS Node API
 // @description Centrifuge OS Node API
-// @version 0.0.5
+// @version 0.0.6
```

Comment:
why the change in version? Did we break anything from the old APIs

Reply:
0.0.5 is what we released as part of Eunomia, and we have been overwritting all over, since the release (pushing to swaggerhub). We need to be more careful when cutting the release to change the version on the first commit after.